### PR TITLE
Add redis payloads

### PR DESCRIPTION
**Pull request by haddasbronfman**

### CODE REVIEW

**Review by osherv — CHANGES_REQUESTED** [review]

File: test/instrumentatios/extentions/redis.test.ts [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+import { RedisInstrumentation } from '@opentelemetry/instrumentation-redis';
+import { configureRedisnstrumentation } from '../../../src/instrumentations/extentions/redis';
+import { Options } from '../../../src';
+import { RedisResponseCustomAttributeFunction } from '@opentelemetry/instrumentation-redis/build/src/types';
+
+diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.ALL);
+
+const instrumentation = new RedisInstrumentation();
+instrumentation.enable();
+instrumentation.disable();
```
[the remark this answers is not in the document]
Comment (haddasbronfman) [reply test/instrumentatios/extentions/redis.test.ts:40]: ok. but it makes the span[] from length 2 to 1. 
so I changes it on test: 'test 2 response hooks'

File: src/instrumentations/index.ts [hunk 2]
```diff
@@ -20,8 +20,8 @@ import { getNodeAutoInstrumentations } from '@opentelemetry/auto-instrumentation
 import { AwsInstrumentation } from '@opentelemetry/instrumentation-aws-sdk';
 import { diag } from '@opentelemetry/api';
 import { configureHttpInstrumentation } from './extentions/http';
-import { configureAWSInstrumentation } from './extentions/aws/aws_sdk';
-import { configureRedisnstrumentation } from './extentions/redis';
+import { configureAwsInstrumentation } from './extentions/aws/aws_sdk';
+import { configureRedisnstrumentation as configureRedisInstrumentation } from './extentions/redis';
```
[the remark this answers is not in the document]
Comment (haddasbronfman) [reply src/instrumentations/index.ts:24]: done

File: package.json [hunk 3]
```diff
@@ -49,6 +50,8 @@
     "@opentelemetry/sdk-node": "^0.27.0",
     "@opentelemetry/sdk-trace-base": "~1.0.1",
     "@opentelemetry/sdk-trace-node": "~1.0.1",
+    "assert": "^2.0.0",
+    "redis": "^3.0.0",
```
Comment (osherv) [inline package.json:54]: Should be dev dependency

File: src/instrumentations/extentions/redis.ts [hunk 4]
```diff
@@ -66,21 +66,25 @@ function createRedisResponseHook(
...
+      case 'del':
+      case 'hgetall':
+      case 'hincrby':
+      case 'exec':
+        span.setAttribute('db.command.arguments', JSON.stringify(cmdArgs));
+        span.setAttribute('db.command.response', JSON.stringify(responseInfo));
         break;
-      }
+      case 'multi':
+        span.setAttribute('db.command.response', responseInfo as string);
```
Comment (osherv) [inline src/instrumentations/extentions/redis.ts:87]: I think the default behavior should be parse arguments and responses as JSON
And check only for 'multi' case, So if we were missed some commands or new commands were developed we will automatically support them

**Review by osherv — APPROVED** [review]

**Pull request merged**